osu.Game/Tests/Visual/OsuTestScene.cs

```diff
@@ -136,7 +146,8 @@ public virtual void RecycleLocalStorage()
         [BackgroundDependencyLoader]
         private void load(RulesetStore rulesets)
         {
-            Ruleset.Value = rulesets.AvailableRulesets.First();
+            if (!Ruleset.Disabled)
+                Ruleset.Value = rulesets.AvailableRulesets.First();
```

Comment:
When can this be disabled? Not sure this should just silently keep going whenever it is, and the old code didn't care either...

Think it might be a vestige of the old now-removed disabling logic as it was alongside it in 67bd7bfa3905aa96f21f2225a517e2e369e80540.

Reply:
oh right, I actually thought it stayed like this before this PR, definitely should be removed.